tests/async_concurrency.rs

```diff
@@ -3,22 +3,18 @@ use graph_rs_sdk::http::GraphResponse;
 use serde::Deserialize;
 use serde::Serialize;
 use test_tools::oauthrequest::OAuthTestClient;
+use graph_rs_sdk::macros::*;
 
 #[derive(Debug, Serialize, Deserialize)]
+#[graph_rs_json]
```

Comment:
We can probably name this better. Something more related to what it does. `odata_next_link` possibly.